README.md

```diff
@@ -17,6 +17,53 @@ The new version number should use the following format.
     Patch: Backwards compatible bug fixes only
     Suffix (optional): a hyphen followed by a string denoting a pre-release version
 
+## Using the package
+The GitHub Nuget Package Repository cannot be accessed anonymously, meaning that to use it in your projects you must add or update the 
+NuGet.Config file to include an authorisation token.
+* If there isn't one there already, create a file in the root of the repository called NuGet.Config.
+* In the NuGet.Config file, add a package source for `github-hackney` that uses the endpoint https://nuget.pkg.github.com/LBHackney-IT/index.json Add or update the `packageSourceCredentials` section to include the credentials for the `github-hackney` source. 
+The token value specified here is a public token whose only permissions are read-only access to the Github Nuget Package Repository.
+```xml
+  <packageSourceCredentials>
+    <github-hackney>
+      <add key="Username" value="PublicToken" />
+      <add key="ClearTextPassword" value="&#103;hp_H1gYc0BCfSXMlf3vRuhHa3lJrxi4Kn4HhR4o" />

```

Comment:
I wouldn't have your public token present in this otherwise other people will start to use it